### ISSUE TITLE
Dropdown aria-live does not work when in Popup with a trap zone or in Dialog

### ISSUE THREAD

**Issue by jurokapsiar** [microsoft/fluent-ui-react#908]
## Bug Report

Dropdown aria-live does not work when in Popup with a trap zone or in Dialog

### Steps
Create an example with Dropdown inside of the Popup trap zone.
Observe aria-live.

### Expected Result
Amounts of available results are narrated

### Actual Result
Amounts of available results are not narrated.
Aria live regions are hidden by aria-hidden by the popup trap focus mechanism. These regions should be excluded.

### Version
x.y.z

### Testcase
[Fork, update, and replace this pen to show the bug]:
https://codesandbox.io/s/2l3n74j9y

**Comment by kuzhelov** [comment]
@jurokapsiar,  may I ask you to share a markup example of the case which repros the problem. Also, please, do not forget to mention the Stardust version in the issue's description. Thank you!

**Comment by sophieH29** [comment]
> @jurokapsiar, may I ask you to share a markup example of the case which repros the problem. Also, please, do not forget to mention the Stardust version in the issue's description. Thank you!

@kuzhelov I can help on this. Aria-live region is usually appended to body, and it's content is narrated by screen reader once content is updated. As you may know, when focus trap zone is rendered (with Popup), it sets `aria-hidden="true"` to all body's children except Popup, so screen reade can't narrate anything outside of Popup. Although, aria-live regions should remain visible to screen reader at any time, as there can be very important messages for users.
See the markup example of what was happening below.

```html
<body>
    <div aria-live="polite" aria-hidden="true"> I should't be hidden from screen reader,
      I need always notify people!</div>
    <div focus-trap-zone>
      <Popup>
   </div>
</body>
```

Let me know if you have any othe questions.
